src/lib/bcmp/dfu/bm_dfu_core.cpp

```diff
@@ -7,28 +7,28 @@
 #include "bm_dfu.h"
 #include "bm_dfu_client.h"
 #include "bm_dfu_host.h"
+#include "bm_dfu_generic.h"
 #include "task_priorities.h"
 #include "device_info.h"
 #include "lpm.h"
 
-using namespace cfg;
+extern "C" {
+#include "messages.h"
+#include "packet.h"
+}
 
 typedef struct dfu_core_ctx_t {
-    libSmContext_t sm_ctx;
+    LibSmContext sm_ctx;
```

Comment:
use the new lib_state_machine.h from bm_common that has the updated style guides.